owl/EASE-OBJ.owl

```diff
@@ -1876,6 +1885,16 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#FunctionalDesign -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#FunctionalDesign">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Design"/>
+        <rdfs:comment>The design of an object from functionality point of view. A functional design is useful to develop complex modular devices which assures that each modular component has specific purpose and can function with minimum side effect on other components of that device. </rdfs:comment>
```

Comment:
"to develop complex modular objects *with components that have a specific purpose* **,** *and that* can function with ...."